benchmarks/versus-external-pools/BENCH-100000.md

```diff
@@ -1,9 +1,13 @@
-| Command                                              |        Mean [s] | Min [s] | Max [s] |    Relative |
-| :--------------------------------------------------- | --------------: | ------: | ------: | ----------: |
-| `node dynamic-piscina.js`                            |  47.192 ± 0.607 |  46.774 |  48.804 | 1.07 ± 0.01 |
-| `node fixed-piscina.js`                              |  47.046 ± 0.112 |  46.823 |  47.178 | 1.07 ± 0.00 |
-| `node dynamic-poolifier.js`                          |  44.301 ± 0.989 |  43.718 |  46.994 | 1.00 ± 0.02 |
-| `node fixed-poolifier.js`                            |  44.115 ± 0.174 |  43.843 |  44.510 |        1.00 |
-| `node static-suchmokuo-node-worker-threads-pool.js`  |  48.282 ± 1.857 |  46.361 |  50.268 | 1.09 ± 0.04 |
-| `node dynamic-suchmokuo-node-worker-threads-pool.js` |  60.111 ± 2.401 |  57.752 |  63.011 | 1.36 ± 0.05 |
-| `node threadjs.js`                                   | 131.412 ± 4.210 | 122.872 | 138.506 | 2.98 ± 0.10 |
+| Command | Mean [s] | Min [s] | Max [s] | Relative |
+|:---|---:|---:|---:|---:|
+| `node dynamic-piscina.js` | 18.364 ± 0.203 | 18.050 | 18.737 | 1.07 ± 0.02 |
+| `node fixed-piscina.js` | 18.222 ± 0.519 | 17.529 | 19.164 | 1.06 ± 0.03 |
+| `node dynamic-poolifier.js` | 17.135 ± 0.256 | 16.926 | 17.755 | 1.00 |
+| `node fixed-poolifier.js` | 17.947 ± 0.489 | 17.567 | 18.951 | 1.05 ± 0.03 |
+| `node dynamic-suchmokuo-node-worker-threads-pool.js` | 25.460 ± 0.453 | 25.124 | 26.617 | 1.49 ± 0.03 |
+| `node static-suchmokuo-node-worker-threads-pool.js` | 18.993 ± 0.714 | 18.388 | 20.610 | 1.11 ± 0.04 |
+| `node threadjs.js` | 80.939 ± 3.548 | 76.881 | 88.363 | 4.72 ± 0.22 |
+| `node dynamic-workerpool.js` | 19.292 ± 0.110 | 19.109 | 19.480 | 1.13 ± 0.02 |
+| `node fixed-workerpool.js` | 19.684 ± 0.636 | 19.212 | 21.049 | 1.15 ± 0.04 |
+| `node fixed-threadwork.js` | 30.023 ± 0.362 | 29.524 | 30.727 | 1.75 ± 0.03 |
+| `node fixed-microjob.js` | 32.526 ± 0.649 | 31.696 | 33.687 | 1.90 ± 0.05 |
```

Comment:
```suggestion
| Command                                              |       Mean [s] | Min [s] | Max [s] |    Relative |
|:-----------------------------------------------------|---------------:|--------:|--------:|------------:|
| `node dynamic-piscina.js`                            | 18.364 ± 0.203 |  18.050 |  18.737 | 1.07 ± 0.02 |
| `node fixed-piscina.js`                              | 18.222 ± 0.519 |  17.529 |  19.164 | 1.06 ± 0.03 |
| `node dynamic-poolifier.js`                          | 17.135 ± 0.256 |  16.926 |  17.755 |        1.00 |
| `node fixed-poolifier.js`                            | 17.947 ± 0.489 |  17.567 |  18.951 | 1.05 ± 0.03 |
| `node dynamic-suchmokuo-node-worker-threads-pool.js` | 25.460 ± 0.453 |  25.124 |  26.617 | 1.49 ± 0.03 |
| `node static-suchmokuo-node-worker-threads-pool.js`  | 18.993 ± 0.714 |  18.388 |  20.610 | 1.11 ± 0.04 |
| `node threadjs.js`                                   | 80.939 ± 3.548 |  76.881 |  88.363 | 4.72 ± 0.22 |
| `node dynamic-workerpool.js`                         | 19.292 ± 0.110 |  19.109 |  19.480 | 1.13 ± 0.02 |
| `node fixed-workerpool.js`                           | 19.684 ± 0.636 |  19.212 |  21.049 | 1.15 ± 0.04 |
| `node fixed-threadwork.js`                           | 30.023 ± 0.362 |  29.524 |  30.727 | 1.75 ± 0.03 |
| `node fixed-microjob.js`                             | 32.526 ± 0.649 |  31.696 |  33.687 | 1.90 ± 0.05 |
```

Reply:
It's the default hyperfine output. It does not worth the effort to reformat it manually like I've done before, it gonna be overwritten each time.